tfjs-backend-webgpu/README.md

```diff
@@ -81,14 +81,8 @@ yarn build
 ```
 
 ## Testing
-The `CHROME_BIN` environment variable must be set to the location of the Chrome
-Canary application.
-
-### Windows
-`set CHROME_BIN=%LOCALAPPDATA%\Google\Chrome SxS\Application\chrome.exe`
-
-### macOS
-`export CHROME_BIN="/Applications/Google Chrome Canary.app/Contents/MacOS/Google Chrome Canary"`
+Currently the Canary channel of Chrome is used for testing of the WebGPU
```

Comment:
I prefer add a reminder here to declare `CHROME_CANARY_BIN` as the ENV_CMD of chrome canary, in case some user wants to test in a specific version of canary.

Reply:
Added a comment below. Hope that's informative enough.

Reply:
For MacOS, that's enough.
For Windows, `yarn test` depends on WSL, `--test_env=CHROME_CANARY_BIN=/path/to/chrome` is not optional but a must, unless we pass CHROME_CANARY_BIN env variable by a custom `.bazelrc.user` config (like https://github.com/tensorflow/tfjs/blob/master/DEVELOPMENT.md#developing-on-windows).
Maybe it's better to divide the comment into 2 parts:
### Windows Subsystem for Linux (WSL)
yarn test --test_env=CHROME_CANARY_BIN=/path/to/chrome
### macOS
yarn test # --test_env=CHROME_CANARY_BIN=/path/to/chrome

Reply:
I'd argue the case of WSL is analogous to "Chrome seems unavailable in your system because it's not found in the well-known paths. It's your bad if it's installed somewhere else, without configuring your distro properly. You should create a symlink in /usr/bin, or just use the CHROME_CANARY_BIN environment variable". WSL is a Frankenstein to me, so I'm against treating it differently than other Unices.

Reply:
Thanks for the explanation, jiajie.
I thought more or less we have a divergence point at "Whether Chrome locates at well-known path".
My point is, even if the Chrome is in well-known paths(default ones which SHOULD be found by karma tools) or symlink in /usr/bin, or exported CHROME_CANARY_BIN in WSL shell, the Karma(started in bazel server) still can not find the chrome start location(in WSL) unless we passed a `--test_env=CHROME_CANARY_BIN` explicitly when bazel server is starting.
e.g. 
In pure Linux, export CHROME_BIN is enough:
`export CHROME_BIN=/path/to/chrome && yarn test`
In WSL, besides CHROME_BIN, we need explicitly pass CHROME_BIN to bazel
`export CHROME_BIN=/path/to/chrome && yarn test --test_env=CHROME_BIN="$CHROME_BIN"`

I agreed that you said WSL is a Frankenstein, maybe WSL made bazel server environment lost "CHROME_BIN" or the default CHROME locations. But it's not the user's fault to make Chrome unavailable and I don't think the user should have the knowledge of "passing environment in WSL when bazel server starts karma". So I thought maybe it's good to write it down and tell user what should to do before a test.

Reply:
I don't think `export CHROME_BIN=/path/to/chrome && yarn test` works as expected in bare metal Linux. That's exactly how the original issue was spotted, as my chrome was not installed in well-known paths and `--test_env=CHROME_BIN` is mandatory in bare metal Linux just as WSL.

I insist that it's users' responsibility to figure out the extra homework if they choose the "wrong" distro (WSL here), so I tend to avoid highlighting this in doc. I do not oppose anyone else adding that back, though.

Reply:
I will merge this one. We can always do more changes later on. Thanks.